rm_reporting/resources/responses_dashboard.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime
+import logging
+
+from flask import json, Response
+from flask_restplus import Resource, abort
+from sqlalchemy import text
+from structlog import wrap_logger
+
+from rm_reporting import app, response_dashboard_api
+from rm_reporting.common.validators import parse_uuid
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@response_dashboard_api.route('/<collection_exercise_id>')
```

Comment:
Might be worth adding collection-exercise in the route. `@response_dashboard_api.route('/collection-exercise/<collection_exercise_id>')`

Reply:
Makes sense for clarity